frontend/src/components/DefenceBox/DefenceMechanism.css

```diff
@@ -1,106 +1,164 @@
-.defence-mechanism {
-	cursor: default;
+.defence-mechanism-fieldset {
+	min-height: fit-content;
+	margin: 0.1em;
+	border: 0.25rem solid var(--control-config-border);
```

Comment:
Great idea to add the border! Makes it clear where one config ends and the next begins :)

```suggestion
	border: 0.15rem solid var(--control-config-border);
```
I'd prefer a slightly less chonky border, but that's a really minor aesthetic choice, so feel free to ignore

Reply:
In my last PR Chris made the point that it would be better to try to use whole-pixel equivalents of rems:
![image](https://github.com/ScottLogic/prompt-injection/assets/125262707/bbdfc923-a06b-401c-b045-5f73c46d0c19)
I tried a skinnier border but thought this looked better chunky.

Reply:
fair enough :)